src/ric/d_00000.c

```diff
@@ -0,0 +1,17 @@
+#include "ric.h"
+
+void UpdateEntityRichter(void);
+void func_80156F40(s16 arg0);
+void func_801603C4(void);
+void func_8015E7B4(Unkstruct_8010BF64* arg0);
+
+PlayerOvl RichterBelmont[] = {
+    /* 0x00 */ UpdateEntityRichter,
+    /* 0x04 */ func_80156F40,
+    /* 0x08 */ func_801603C4,
+    /* 0x0C */ func_8015E7B4,
+};
+
+s32* D_8013C010[4] = {
+    &D_8013C010[LEN(D_8013C010)], &D_8013C010[LEN(D_8013C010)],
+    &D_8013C010[LEN(D_8013C010)], &D_8013C010[LEN(D_8013C010)]};
```

Comment:
I believe the symbol `g_richter_spritesheet` resolves to `0x8013C020` (i.e. `&D_8013C010[LEN(D_8013C010)]`) at build time.

Reply:
It can't be used because g_richter_spritesheet needs to be defined in symbols.us.ric.txt for the linker to even know which address it corresponds to; which we did, at first, but the linter removes it from the txt file, so we were instructed by Xeeynamo to do the above